app/controllers/grant_users_controller.rb

```diff
@@ -0,0 +1,93 @@
+class GrantUsersController < ApplicationController
+	before_action :set_grant, only: %i[index new edit create update destroy]
+	before_action :set_grant_user, only: %i[edit update destroy]
+
+	# GET /grants/:study_id/grant_users
+	def index
+		@grant_users = @grant.grant_users.all
+		authorize @grant_users
+	end
+
+	# GET /grants/:study_id/grant_user/new
+  def new
+		@emails = unassigned_grant_users_by_organization.pluck(:email)
```

Comment:
Consider getting user id and email. Then you can:

- Use options_for_select method to populate the drop down in the view; and
- Pass the user id from the form rather than needing to look it up later in the create action.